js/navigation-search.js

```diff
@@ -5,20 +5,18 @@ bcpl.navigationSearch = (($) => {
 	const searchBoxSelector = '#search-box';
 	const searchButtonSelector = '#search-button';
 	const hamburgerButtonSelector = '#hamburger-menu-button';
-	const menuSelector = '.nav-and-search nav';
-	const navBackButtonSelector = '.nav-back-button button';
+	const menuSelector = '#responsive-sliding-navigation';
+	const navBackButtonSelector = '#responsive-sliding-navigation > .nav-back-button button';
 	const modalCoverSelector = '#modal-cover';
-	const menuItemsSelector = '.nav-and-search nav > ul > li > button';
-	const submenuBackButtonSelector = '.nav-and-search nav ul li ul li button';
 
 	/* Helpers */
 
 	const killMenuAndModalCover = ($menu, $modalCover) => {
 		$modalCover.removeClass('active');
```

Comment:
should we take this approach here? https://stackoverflow.com/a/8526263/1143670

Reply:
Good idea, though I removed those lines so I'm not sure why you're seeing them.